docs/working-with-velero.md

```diff
@@ -1,25 +1,25 @@
 # Integrate Velero to back up and restore Karmada resources
 
-[Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore 
-your Kubernetes cluster resources and persistent volumes. You can run Velero with a public
-cloud platform or on-premises.
+- [Velero](https://github.com/vmware-tanzu/velero) gives you tools to back up and restore.
+- Your Kubernetes cluster resources and persistent volumes. You can run Velero with a public.
+- Cloud platform or on-premises.
```

Comment:
Why?

Reply:
Sorry, my branch is a little messy. I'll close this submission first and resubmit later. I have read this sentence carefully and will not make any additional modifications.